src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+  border-radius: 5px;
+  background: #fff;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  transition: 0.3s;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
```

Comment:
its bad way to use fix margin for center img use flex or margin auto

Reply:
I know that it is a bad way, but it is a card and tests demand hardcoded positioning.